components/about_build_modal/about_build_modal.jsx

```diff
@@ -12,6 +12,8 @@ import Nbsp from 'components/html_entities/nbsp';
 
 import {AboutLinks} from 'utils/constants';
 
+import AboutBuildModalCloud from './about_build_modal_cloud/about_build_modal_cloud';
```

Comment:
For future reference, if you're not connecting the component to redux, you don't need to create a subfolder for it.